website/_includes/manual/connect-using-file.md

```diff
@@ -23,3 +23,7 @@ Here is an example:
     password=
     #optionally specify a database name
     database=dbfit
+
+### Second connection using `ConnectUsingFile`
+
+If you open second connection it advised to use command |close|.
```

Comment:
Instead of new section (###) I think this may be formatted like:

```
<div class="alert alert-warning alert-block">
If you need to open a second connection it's advised to close the current one first using <code>|Close|</code> command.
</div>
```